src/libtiled/imagelayer.h

```diff
@@ -114,6 +114,26 @@ class TILEDSHARED_EXPORT ImageLayer : public Layer
      */
     bool isEmpty() const override;
 
+    /**
+     * Returns true if the image of this layer repeats horizontally.
+     */
+    bool repeatHorizontally() const { return mRepeatHorizontally; }
```

Comment:
Let's go with a shorter forms like in CSS, `repeatX` and `repeatY`.

Reply:
Done.